javascript/javascript3/week1/weatherApp/weatherApp.js

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
In general really good job on the weather app here! I like the naming of your functions and variables as it makes it quite easy to read and understand. I only have a few comments on things that could be even better. 😃